app/styles/partials/utils.scss

```diff
@@ -10,3 +10,19 @@
 .attendees-margin {
   margin: 5px 0 5px 0 !important;
 }
+.public {
+  height: 100vh;
+  overflow: hidden !important;
+}
+.add-order {
+  padding-top: 20px !important;
+}
+.field.payments {
+  margin-top: 10px;
+}
+.content.tax-info {
```

Comment:
These are all specific to a component. What is this doing in the file for utility classes ?